torchvision/models/detection/transform.py

```diff
@@ -30,25 +30,33 @@ def _resize_image_and_masks(image, self_min_size, self_max_size, target):
     else:
         im_shape = torch.tensor(image.shape[-2:])
 
-    min_size = torch.min(im_shape).to(dtype=torch.float32)
-    max_size = torch.max(im_shape).to(dtype=torch.float32)
-    scale = torch.min(self_min_size / min_size, self_max_size / max_size)
-
-    if torchvision._is_tracing():
-        scale_factor = _fake_cast_onnx(scale)
+    if self_min_size == self_max_size:  # TODO: Improve this workaround
+        # Fixed size output. Assume width / height the same.
```

Comment:
The assumption that width == height when the size is fixed does not hold always. For example we might have a model that requires fixed input of size 100x200. This is a quick and dirty patch to kick off training (SSD uses 300x300 input), but it needs to be revised.